storage/pipeline/states_sealing.go

```diff
@@ -213,6 +215,40 @@ func (m *Sealing) handleGetTicket(ctx statemachine.Context, sector SectorInfo) e
 	})
 }
 
+var SoftErrRetryWait = 5 * time.Second
+
+func retrySoftErr(ctx context.Context, cb func() error) error {
+	for {
+		err := cb()
+		if err == nil {
+			return nil
+		}
+
+		var cerr storiface.WorkError
+
+		if errors.As(err, &cerr) {
+			switch cerr.ErrCode() {
+			case storiface.ErrTempWorkerRestart:
```

Comment:
We should probably define all errors we consider "soft errors" somewhere above and add a comment on the error enum to evaluate if a newly added error is a soft error and if so, add it to that list

Reply:
That's what the `Temp` in the error name stands for